lib/config/configuration.go

```diff
@@ -2146,12 +2146,10 @@ func applyTokenConfig(fc *FileConfig, cfg *service.Config) error {
 	if fc.AuthToken != "" {
 		cfg.JoinMethod = types.JoinMethodToken
 		cfg.SetToken(fc.AuthToken)
-
-		return nil
```

Comment:
removing the early return here so that we'll actually catch cases where `auth_token` and `join_params` are both set.